functions/version-policy.js

```diff
@@ -18,7 +18,7 @@ function checkPaths(targetVal) {
       return [
         {
           message: `Version segment "${version}" in basePath violates Azure versioning policy.`,
-          path: basePath,
+          path: [basePath],
```

Comment:
```suggestion
          path: [String(basePath)],
```
you may want to be even stricter if needed.

Reply:
THANK YOU for finding this bug!  I will fix right away.